fjord/base/utils.py

```diff
@@ -279,7 +279,7 @@ def add_item_elements(self, handler, item):
                 attrs={'href': item['link_related'], 'rel': 'related'})
 
 
-def actual_ip(req):
+def actual_ip(group, req):
```

Comment:
django-ratelimit changed the function signature for key functions from `(request)` to `(group, request)`.